pkg/controller/federatedcluster/controller.go

```diff
@@ -233,28 +233,26 @@ func thresholdAdjustedClusterStatus(clusterStatus *fedv1a1.FederatedClusterStatu
 	return clusterStatus
 }
 
-func updateClusterZoneAndRegion(clusterStatus *fedv1a1.FederatedClusterStatus, clusterSpecifics *ClusterData,
+func updateClusterZoneAndRegion(clusterStatus *fedv1a1.FederatedClusterStatus, cluster *fedv1a1.FederatedCluster,
 	clusterClient *ClusterClient) *fedv1a1.FederatedClusterStatus {
 
 	if !util.IsClusterReady(clusterStatus) {
 		return clusterStatus
 	}
 
-	zone, region, err := clusterClient.GetClusterZones()
+	zones, region, err := clusterClient.GetClusterZones()
 	if err != nil {
 		glog.Warningf("Failed to get zones and region for cluster %q: %v", clusterClient.clusterName, err)
```

Comment:
(No action required) Consider returning after this warning to remove the need for protecting the non-error behavior with the else clause.

Reply:
Done.